packages/gatsby/src/utils/feedback.ts

```diff
@@ -111,3 +133,38 @@ export async function userPassesFeedbackRequestHeuristic(): Promise<boolean> {
   // for feedback
   return true
 }
+
+function isFeedbackDisabled(): boolean {
+  // Heuristic 2
+  if (getConfigStore().get(feedbackKey) === true) {
+    return true
+  }
+
+  // Heuristic 3
+  if (process.env.GATSBY_FEEDBACK_DISABLED === `1`) {
+    return true
+  }
+  return false
+}
+
+export async function userGetsSevenDayFeedback(): Promise<boolean> {
+  if (isCI || isFeedbackDisabled()) return false
```

Comment:
I'm not 100% sure but isn't isCI a function?
```suggestion
  if (isCI() || isFeedbackDisabled()) return false
```

Reply:
Shouldn't we add isCi() to the isFeedbackDisabled call instead?

Reply:
We certainly can. That's shared between both calls and right now (for some reason) we don't check CI for the regular feedback request. If we're good to change that behavior then I'll add it.